.gitignore

```diff
@@ -15,23 +15,36 @@
 # downloaded files during tests with cypress
 /cypress/downloads
 
+# next.js
+/.next/
+/out/
+
 # production
 /build
 
-# ide
-.idea
-!.idea/dictionaries/hedgedoc.xml
-!.idea/copyright
-!.idea/prettier.xml
 # misc
 .DS_Store
+*.pem
+
+# debug
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# local env files
 .env.local
 .env.development.local
 .env.test.local
 .env.production.local
 
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
+# vercel
+.vercel
+
+# typescript
+*.tsbuildinfo
 
-.eslintcache
```

Comment:
Do we want to track .eslintcache?

Reply:
This file isn't used anymore for a long time.

Reply:
The file now resides in `node_modules/.cache/.eslintcache`. This path is already covered by the gitignore file.